examples/03_connectivity/plot_signal_extraction.py

```diff
@@ -73,9 +74,86 @@
 plotting.plot_matrix(correlation_matrix, figure=(10, 8), labels=labels[1:],
                      vmax=0.8, vmin=-0.8, reorder=True)
 
+##############################################################################
+# Load confounds from file using a flexible strategy with load_confounds
+# ----------------------------------------------------------------------
+# The :mod:`nilearn.load_confounds` module can be used to create
+# a :class:`nilearn.load_confounds.Confounds` class with flexible
+# parameters. We create a :class:`nilearn.load_confounds.Confounds` class
+# which specifies strategies and optional parameters. Then the
+# :func:`nilearn.load_confounds.Confounds.load` method selects the relevant
+# columns from the TSV file. Let's try a strategy similar to Params9 from
+# Ciric et al. 2017, and some of its variations:
+
+from nilearn.load_confounds import Confounds
+params9 = Confounds(strategy=["high_pass", "motion", "wm_csf", "global"],
```

Comment:
It was simply the one a previous demo used. Still I think it's a good one for showcasing: it's a fairly standard motion based strategy.

Reply:
I see, but the naming is bad: it is unclear what "9" refers to.

Reply:
This name is derived from Ciric et al. 2017 - they call it "9P": 6 motion parameters + average CSF, WM and global signal (9 confounds total + the high pass filter). Maybe it would be better to stick with the terminology we'll use for strategies in Nilearn. This would be `Minimal` and a global signal, and we can mention in passing this fits with the `9P` from Ciric. WDYT @htwangtw ?

Reply:
That sounds like a sensible idea.